### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by didukhroma**
add privacy policy html
add terms html
feature fixed footer
feature subscribe logic

### CODE REVIEW

File: pullrequests-list.txt [hunk 1]
```diff
@@ -1 +1 @@
-
+https://github.com/asaulyak/a-team-advancedjs/pull/5
```
Comment (asaulyak) [inline pullrequests-list.txt:1]: remove this file
it supposed to be excluded

File: src/fixedFooter/fixedFooter.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import throttle from 'lodash.throttle';
```
Comment (asaulyak) [inline src/fixedFooter/fixedFooter.js:1]: stick footer to the bottom with css
use min-height for the main content

File: pullrequests-list.txt (hunk 1 above)
Comment (didukhroma) [reply pullrequests-list.txt]: deleted

File: src/fixedFooter/fixedFooter.js (hunk 2 above)
Comment (didukhroma) [reply src/fixedFooter/fixedFooter.js]: add style for main

**Review by asaulyak — APPROVED** [review]

**Pull request merged**